### ISSUE TITLE
Puppeteer hangs for some websites (Navigation Timeout Exceeded: 30000 ms exceeded.)

### ISSUE THREAD

**Issue by deinega** [puppeteer/puppeteer#2381]
Puppeteer hangs at some websites, throwing following error:
pyppeteer.errors.TimeoutError: Navigation Timeout Exceeded: 30000 ms exceeded.

This is the example:
```
async def main():
    browser = await launch({'headless': False})
    page = await browser.newPage()
    await page.goto('https://www.ferragamo.com/shop/us/en/women/silk-bijoux/shawls-and-stoles')
    await browser.close()
asyncio.get_event_loop().run_until_complete(main())
```

I have tried 'waitUntil': 'networkidle0', 'waitUntil': 'networkidle2'. It does not help for this particular website.

**Comment by aslushnikov** [comment]
@deinega What version do you use? Puppeteer 1.3 works just fine for me:

```js
const puppeteer = require('puppeteer');

(async () => {
  const browser = await puppeteer.launch();
  const [page] = await browser.pages();
  await page.goto('https://www.ferragamo.com/shop/us/en/women/silk-bijoux/shawls-and-stoles');
  await browser.close();
})();
```

**Comment by deinega** [comment]
Thank for you reply.

I am actually using python version of puppeteer called pyppeteer (version 0.0.17):
https://github.com/miyakogi/pyppeteer

According to their documentation, it is based on 1.0.0:
https://miyakogi.github.io/pyppeteer/changes.html#version-0-0-17-2018-04-02

**Comment by panthony** [comment]
@deinega If it works using the 1.3 then your issue was probably already fixed since the 1.0.0.

You should probably create an issue to the other repository asking them to update their version of puppeteer instead.

**Comment by kireerik** [comment]
I am using version `1.3.0` on Heroku and sometimes I get this error too through Sentry error reporting.

I am using Puppeteer directly with Node.js `9.x`.
